doc/book/src/user/installation.md

```diff
@@ -0,0 +1,84 @@
+# Installation
+
+## Debian / Ubuntu
+
+The Electric Coin Company operates a package repository for 64-bit Debian-based distributions.
+
+### Fetching a New Signing Key
+
+First install the following dependency so you can talk to our repository using HTTPS:
+
+```
+sudo apt-get update && sudo apt-get install apt-transport-https wget gnupg2
+```
+
+Next add the Zcash master signing key to apt’s trusted keyring:
+
+```
+wget -qO - https://apt.z.cash/zcash.asc | gpg --import
+gpg --export B1C9095EAA1848DBB54D9DDA1D05FDC66B372CFE | sudo apt-key add -
```

Comment:
I have not verified this key handle.

Reply:
I calculated this with these commands:

```
$ wget -qO - https://apt.z.cash/zcash.asc | gpg --import
gpg: key 1D05FDC66B372CFE: "Zcash Master Signing Key (Electric Coin Company) <sysadmin@z.cash>" not changed
gpg: Total number processed: 1
gpg:              unchanged: 1

$ gpg --fingerprint 1D05FDC66B372CFE
pub   rsa3072 2023-06-19 [SC]
      B1C9 095E AA18 48DB B54D  9DDA 1D05 FDC6 6B37 2CFE
uid           [ unknown] Zcash Master Signing Key (Electric Coin Company) <sysadmin@z.cash>
sub   rsa3072 2023-06-19 [E]

$ gpg --fingerprint 1D05FDC66B372CFE | grep 2CFE | sed 's/ *//g'
B1C9095EAA1848DBB54D9DDA1D05FDC66B372CFE
```

So this is relying on `apt.z.cash` (and the wget/tls/gpg toolchain). We should verify this matches the debian signing keypair we use for deploying those packages.

Reply:
The key used is the correct one, it is the one used for debian package releases.